services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1464,6 +1464,7 @@
             },
             "dashboardTitle": "Report program characteristics & enrollment for each plan",
             "drawerTitle": "Report program characteristics & enrollment for",
+            "drawerReminder": "Complete all fields and select the Save & close button to save this section.",
             "missingEntityMessage": [
```

Comment:
nit: should it be `Save & Close` instead of `Save & close`

Reply:
the ticket had it listed as lowercase but uppercase would be more cohesive. i'm not sure who to ask for preference